src/frontend/src/store/index.ts

```diff
@@ -366,5 +373,21 @@ export default new Vuex.Store({
         dismissible: true,
       });
     },
+
+    ACTION_FETCH_TRANSACTION_HISTORY({ commit, getters, dispatch }) {
+      axios
+        .get("http://localhost//wallet/history/"+ getters.umnetId)
```

Comment:
Extra '/` here in `localhost//wallet`, not sure if it actually affects the success of the call or not